### PR TITLE
Fix/gurobi version

### LINKED ISSUE #245: possible bug in gurobi version
Hi there, 

I am running python `3.10.6` and I just installed gurobi 10.

It is my belief that [line 39](https://github.com/opencobra/optlang/blob/master/src/optlang/gurobi_interface.py#L39) could lead to an issue as: 
```python
>>> import gurobipy
>>> version = gurobipy.gurobi.version()
>>> version
(10, 0, 0)
```
and the if statement in the `gurobi_interface.py` script asks for: 
```    
if not (version[0] >= 9 and version[1] >= 5):
        raise RuntimeError()`
```

Hope this helps and apologies if I have missed something.

### PR DESCRIPTION
**Pull request by cdiener**
* [X] fix #245 
* [X] description of feature/fix
* [X] tests added/passed
* [X] add an entry to the [next release](../CHANGELOG.rst)

Hotfix to fix failures when using Gurobi 10.0 or later.

### CODE REVIEW

File: src/optlang/gurobi_interface.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 try:
     version = gurobipy.gurobi.version()
-    if not (version[0] >= 9 and version[1] >= 5):
+    if not version[:2] >= (9, 5):
```
Comment (Midnighter) [inline src/optlang/gurobi_interface.py:39]: Would it be more straight forward to say
```suggestion
    if version[:2] < (9, 5):
```
?

File: setup.cfg [hunk 2]
```diff
@@ -35,7 +35,7 @@ keywords =
 zip_safe = True
 install_requires =
     six >=1.9
-    swiglpk >=1.4.3
+    swiglpk >=4.65.1
```
Comment (Midnighter) [inline setup.cfg:38]: How about going to 5.x directly?

**Review by Midnighter — APPROVED** [review]
Thank you for the quick fix. Two suggestions which you are in a better position to judge.

File: setup.cfg (hunk 2 above)
Comment (cdiener) [reply setup.cfg:38]: sure

**Pull request merged**